Cargo.toml

```diff
@@ -15,5 +15,5 @@ include = ["src/*", "Cargo.toml", "LICENSE-MIT", "LICENSE-APACHE", "README.md"]
 quick-xml = "0.12"
 derive_builder = "0.5"
 serde = { version = "1.0", optional = true, features = ["derive"] }
-failure = "0.1.1"
-chrono = "0.4.5"
+failure = "0.1"
+chrono = { version = "0.4", features = ["serde"] }
```

Comment:
i _think_, this means results in the `serde` feature on the `chrono` crate is being compiled even if the user doesn't have the `serde` feature enabled on the `atom` crate.

we can work around this by introducing a new `use-serde` feature that will enable `serde` and `chrono/serde`, as seen here:

https://github.com/georust/geo/blob/b57ab9ffac65e0ce45e549c45acc4ef6408fd310/geo/Cargo.toml#L29

does this make sense?

Reply:
another example in the `session` feature in https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section

Reply:
Thanks for the help! I've add a new feature `with-serde`, which declared as `with-serde = ["serde", "chrono/serde"]`, it should resolve this problem.